config/routes.rb

```diff
@@ -76,24 +76,11 @@
     resources :assessment_results, only: %i[show new], path: 'assessment-result'
   end
 
-  # STATIC ---------------------------------------------------------------------
+  # STATIC PAGES ---------------------------------------------------------------
 
-  # to be removed and below used when migrating to contentful
-  static_regexp = %r{
-    accessibility-statement|
-    cookie-policy|
-    new-registration|
-    other-problems-signing-in|
-    privacy-policy|
-    terms-and-conditions|
-    sitemap|
-    settings/cookies|
-    whats-new|
-    wifi-and-data
-  }x
-  get '/:id', to: 'static#show', id: static_regexp, as: :static
-
-  scope module: 'contentful' do
-    resources :static, only: %i[show], as: :static_pages, path: ''
+  if Rails.application.cms?
+    resources :pages, only: %i[show], path: '/', as: :static
```

Comment:
Can you confirm this works?  Does it need to be a proc?

Reply:
The test suite tests this because it runs twice; but I can also confirm I have toggled my dev env back and forth and seen it working. I didn't use constraints because they aren't the same `resource`, but you're right it isn't idiomatic rails routing. I'll go try it now and see if it actually works out the box.

Reply:
- `ArgumentError: Invalid route name, already in use: 'static'` is the blocker if we try to define them twice, first inside the existing constraint block.
- `if proc { Rails.application.cms? }` would always be true and would always load from the CMS.